### ISSUE TITLE
[select ... from ... where ...order by convert(col using gbk); ] get ERROR: CONVERT(... USING ...) is standard SQL syntax

### ISSUE THREAD

**Issue by irene-coming** [actiontech/dble#982]
* **dble version：**  
5.6.29-dble-9.9.9.9-77ac73eb405ba6933bc967e0bdbbfd70fdcf9f1e-20190201043948

* **preconditions ：**  
no
* **configs：** 

**schema.xml**  

```
<table dataNode="dn1,dn2,dn3,dn4" name="sharding_four" rule="hash-four" />
```

* **steps：**    
    step1.  execute sqls with dble:
```mysql
mysql> create table sharding_four(id int, c varchar(10)) charset utf8mb4;
Query OK, 0 rows affected (0.06 sec)

mysql> insert into sharding_four values(1, 'a'),(2, '吃'),(3,'啊'),(4,'吧'),(1004,'吃');                                      Query OK, 5 rows affected (0.00 sec)

mysql> select convert("中" using gbk);
+--------------------------+
| convert("中" using gbk)  |
+--------------------------+
| 中                       |
+--------------------------+
1 row in set (0.04 sec)
```
step2.  execute sqls with dble:
```mysql
mysql> select * from sharding_four where id in (1,2,3,4,1004) order by convert(c using gbk);
ERROR 4004 (HY000): CONVERT(... USING ...) is standard SQL syntax
```

* **expect result：**  
    1. step2 query success
* **real result：**  
    1. step2 query failed
* **supplements：**  
    1. if this issue is fixed , need to verify chinese character order right

/label ~BUG

**Comment by yanhuqing666** [comment]
bf60cb1b4e4d6bbbf6a693935eb0410bab4d051b

**Comment by yanhuqing666** [comment]
bf60cb1b4e4d6bbbf6a693935eb0410bab4d051b

**Comment by yexiaoli88** [comment]
syntax error has been fixed and verified on dble version：5.6.29-dble-9.9.9.9-884fc6b612d64cc22101226536f8fd1d24580857-20190221031624
chinese character order related issue #1011
